contrib/vdirsyncer.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <!-- click requires a proper UTF-8 locale -->
+    <key>EnvironmentVariables</key>
+    <dict>
+           <key>LANG</key>
+           <string>en_US.UTF-8</string>
+           <key>LC_ALL</key>
+           <string>en_US.UTF-8</string>
```

Comment:
So there's generally a lot of confusion around this: Click generally *respects* the locale, whatever it is set to. Python 3 implies that a missing locale configuration implies the `C` locale -- Click treats both conditions as an error, since it's more likely a result of a misconfigured environment (with missing envvars) rather than intention. Unfortunately it can't detect if the C locale is intentionally set, but usually it isn't.

A few questions:

* What is `LANG`/`LC_ALL` set by default in launchd "cronjobs"?
* What are other plists doing?